sql/field.cc

```diff
@@ -4355,6 +4355,20 @@ void Field_longlong::sql_type(String &res) const
   add_zerofill_and_unsigned(res);
 }
 
+bool Field_longlong::set_max()
+{
+  ASSERT_COLUMN_MARKED_FOR_WRITE_OR_COMPUTED;
+  int8store(ptr, ULONGLONG_MAX);
```

Comment:
Storing ULONGLONG_MAX for signed Field_longlong class and reading signed in unsigned variable here
```ulonglong j;
j = sint8korr(ptr);
```
looks strange.